src/demokratie.md

```diff
@@ -133,17 +133,17 @@ wie z. B. heise.de bereits 2005 dokumentiert:
 > kritischer Bürger entzogen bleiben.
 > [Informationsfreiheitsgesetz in der Kritik](https://www.heise.de/newsticker/meldung/Informationsfreiheitsgesetz-in-der-Kritik-144367.html)
 
-Wenn man mit versucht mit der Politik schriftlich zu kommunizieren, ist oft
-nicht klar, wer Entscheidungen getroffen hat, da keine Rechtfertigungspflicht
+Wenn man versucht, mit der Politik schriftlich zu kommunizieren, ist oft
+nicht klar wer Entscheidungen getroffen hat, da keine Rechtfertigungspflicht
```

Comment:
```suggestion
Wenn man versucht mit der Politik schriftlich zu kommunizieren, ist oft
nicht klar wer Entscheidungen getroffen hat, da keine Rechtfertigungspflicht
```

Reply:
"zu kommunizieren" erfordert ein Komma davor. Unter dem Kapitel "Komma vor Infinitivgruppen" des nachfolgenden Links wird das erklärt. https://www.phase-6.de/magazin/rubriken/grammatik/kommaregeln-einfach-erklaert-in-4-schritten/